examples/app-basic/tests/fixtures.ts

```diff
@@ -0,0 +1,19 @@
+import { AppConfigPage, test as base } from '@grafana/plugin-e2e';
+import pluginJson from '../src/plugin.json';
+
+type AppTestFixture = { appConfigPage: AppConfigPage };
+
+export const test = base.extend<AppTestFixture>({
```

Comment:
I like that we are able to define the pluginId once and then can reuse it for all tests.

Reply:
Yes this is really neat 👍